src/Console/Command/RepackCommand.php

```diff
@@ -20,6 +20,7 @@ protected function configure(): void
     {
         $this
             ->setName('repack')
+            ->setAliases(['castor:repack'])
```

Comment:
Should'nt we inverse the name and alias?

Reply:
you are right, and it's now consistant with the debug command